style.css

```diff
@@ -526,41 +600,451 @@
+    margin-left: 10px;
+  }
+
+  .contact-info {
+    height: 100%;
+    width: 45%;
+    background-color: #ffff;
+  }
+
+  .contact2 {
+    background-color: #ffff;
+  }
+
+  .half-width {
+    width: 50%;
+  }
+
+  .social {
+    width: 35%;
+  }
+
+  .about-desktop2 {
+    flex-direction: row;
+    justify-content: flex-start;
+    gap: 30px;
+    padding-inline: 143px;
+  }
+
+  .skills {
+    height: 100%;
+    width: 55%;
+    margin-bottom: 200px;
+  }
+
+  .form-section {
+    background-image: url(./vectors/contact\ form\ background\ shapes.svg);
+    background-size: 90%;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding-inline: 350px;
+  }
+
+  .languages-list {
+    flex-direction: row;
+  }
+
+  .languages-list li {
+    flex-direction: column;
+  }
+
+  .hamburguer-hidden {
+    display: none;
+  }
+
+  .nav-desktop a {
+    background-color: #ffff;
+    text-decoration: none;
+  }
+
+  .card-desktop {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 924px;
+    background-color: #ffff;
+  }
+
+  .grid-container-desktop {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 0 382px;
+    gap: 24px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: auto;
+    background-color: #ffff;
+  }
+
+  .container2-desktop {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .p-about-background {
+    padding: 0;
+  }
+
+  .card-img {
+    width: 120%;
+  }
+
+  .card-img2 {
+    background-color: #ffff;
+  }
+
+  .card1 {
+    display: none;
+  }
+
+  .card-img5 {
+    width: 544px;
+    height: 448px;
+  }
+
+  .card-container-reverse {
+    flex-direction: row-reverse;
+    justify-content: space-around;
+  }
+
+  .about-desktop {
+    display: flex;
+    background: #ffff;
+  }
+
+  .header2 {
+    background-color: #ffff;
+    border: 1px solid #ffff;
+    border-top-right-radius: 43px;
+    display: flex;
+  }
+
+  .languages2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+    margin: 0 8%;
+    list-style: none;
+    background-color: #ffff;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .prueba2 {
+    display: grid;
+    grid-template-rows: repeat (2, 1fr);
+  }
+
+  .title-desktop {
+    width: 60%;
+    background-color: #ffff;
+    margin-left: 132px;
+    margin-top: 39px;
+  }
+
+  .align-center {
+    margin: 0 auto;
+  }
+
+  .about-h1-desktop {
+    margin-left: 10px;
+    height: 10px;
+  }
+
+  .page-background2 {
+    background-size: cover;
+  }
+
+  .works-container {
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form-section p {
+    margin-left: -263px;
+    margin-right: -307px;
+  }
+
+  .form-section h1 {
+    margin-left: -263px;
+    margin-right: -307px;
+  }
```

Comment:
You have done great by working on the contact section as requested by the previous reviewer. However, there's a little more. Kindly ensure that the centre alignment if the elements are maintained.

Your design:
![image](https://user-images.githubusercontent.com/20554583/229209203-db9c07f2-ea2b-4f89-bacb-66422b04c585.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/229209086-14fc6d06-3b4e-4bf5-a85f-bb288821ab27.png)